svi/auto_thermal_reformer/get_block_causing_jump.py

```diff
@@ -0,0 +1,85 @@
+#  ___________________________________________________________________________
```

Comment:
This script has not been superseded by any of my (more) recent commits to this PR.